### PR TITLE
Add Arduino I2C shield type

### PR DESCRIPTION
**Pull request by lbussy**
This merges in changes made in BPR which supports the I2C shield type.

### CODE REVIEW

File: brewpi-script/scriptlibs/pinList.py [hunk 1]
```diff
@@ -41,26 +41,47 @@ def getPinList(boardType, shieldType):
                    {'val': 20, 'text': 'A2', 'type': 'free'},
                    {'val': 21, 'text': 'A3', 'type': 'free'}]
     elif boardType == "uno" and shieldType == "revC":
-        pinList = [{'val': 6, 'text': ' 6 (Act 1)', 'type': 'act'},
-                   {'val': 5, 'text': ' 5 (Act 2)', 'type': 'act'},
+        pinList = [{'val': 0, 'text': ' 0', 'type': 'serial'},
```
Comment (thorrak) [inline brewpi-script/scriptlibs/pinList.py]: What's with the changes to the existing revC pinlist?

File: brewpi-script/scriptlibs/pinList.py [hunk 2]
```diff
@@ -41,26 +41,47 @@ def getPinList(boardType, shieldType):
...
-                   {'val': 0, 'text': ' 0', 'type': 'serial'},
-                   {'val': 1, 'text': ' 1', 'type': 'serial'},
                    {'val': 14, 'text': 'A0', 'type': 'free'},
                    {'val': 15, 'text': 'A1', 'type': 'free'},
                    {'val': 16, 'text': 'A2', 'type': 'free'},
-                   {'val': 17, 'text': 'A3', 'type': 'free'}]
+                   {'val': 17, 'text': 'A3', 'type': 'free'},
+                   {'val': 18, 'text': 'A4 (OneWire)', 'type': 'onewire'},
+                   {'val': 19, 'text': 'A5 (Act 4)', 'type': 'act'}]
+    elif boardType == "uno" and shieldType == "I2C":
```
Comment (thorrak) [inline brewpi-script/scriptlibs/pinList.py]: I'm assuming you stuck with shield type "I2C"? If so, I'll get this merged in.

File: brewpi-script/scriptlibs/pinList.py (hunk 1 above)
Comment (lbussy) [reply brewpi-script/scriptlibs/pinList.py]: I added all pins for consistency/clarity, even those not used.  In BrewPi, "serial" or "free" do not show up as choices in the drop-downs.

File: brewpi-script/scriptlibs/pinList.py (hunk 2 above)
Comment (lbussy) [reply brewpi-script/scriptlibs/pinList.py]: I did.

**Comment by lbussy** [comment]
I was wondering if you were gonna pick up on that. :)

**Pull request merged**